### PR TITLE
Fix aiohttp

### PR DESCRIPTION
**Pull request by Rizato**
Updates aiohttp, and aioresponses. 

Also, includes some gitlab changes to add a few extra tags. `:MAJOR` `:MAJOR.MINOR`, `:MAJOR.MINOR.PATCH`.

### CODE REVIEW

File: src/polyswarmclient/utils.py [hunk 1]
```diff
@@ -22,6 +22,16 @@
 MAX_WAIT = int(os.getenv('WORKER_BACKOFF', '15'))
 MAX_WORKERS = 4
 
+DEPRECATION_MESSAGE = '''
+    polyswarm-client 2.x.x is deprecated.
+    However, polyswarm-client 2.x.x will still work with some PolySwarm communities in the near term.
+
+    Action is required to continue using 2.x.x.
+
+    If your build is based off the polyswarm-client:latest docker image, you MUST change another tag.
```
Comment (alanjds) [inline src/polyswarmclient/utils.py:31]:
```suggestion
    If your build is based off the polyswarm-client:latest docker image, you MUST change to another tag.
```

**Review by alanjds — APPROVED** [review]

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -218,15 +230,27 @@ release-tag-windows:
...
     # Gets the current image that was built in the CI for this commit
     - docker pull "$REPO_URL/$BASE_IMAGE_NAME-windows:$CI_COMMIT_SHA"
+    # Separate tag into multiple parts
+    - TAG=$(git describe --tags --abbrev=0)
+    - MAJOR=`python -c "print('$TAG'.split('.')[0])"`
+    - MINOR=`python -c "print('$TAG'.split('.')[1])"`
     # Creates new tags for this image, one that should go to AWS and another to DockerHub with the tag from git
-    - docker tag "$REPO_URL/$BASE_IMAGE_NAME-windows:$CI_COMMIT_SHA" "$REPO_URL/$BASE_IMAGE_NAME-windows:$(git describe --tags --abbrev=0)"
-    - docker tag "$REPO_URL/$BASE_IMAGE_NAME-windows:$CI_COMMIT_SHA" "polyswarm/$BASE_IMAGE_NAME-windows:$(git describe --tags --abbrev=0)"
+    - docker tag $REPO_URL/$BASE_IMAGE_NAME:$CI_COMMIT_SHA $REPO_URL/$BASE_IMAGE_NAME:$TAG
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml:238]: @Rizato you are missing the `-windows` part of the image name

File: src/polyswarmclient/utils.py [hunk 3]
```diff
@@ -299,3 +299,16 @@ def fill_scheme(polyswarmd_addr, insecure_transport):
...
+
+def fast_deprecation():
+    warnings.simplefilter('module', DeprecationWarning)
+    warnings.warn('''
+                          polyswarm-client 2.x.x is deprecated.
+                          However, polyswarm-client 2.x.x will still work with some PolySwarm communities in the near term.
+
+                          Action is required to continue using 2.x.x.
+
+                          If your build is based off the polyswarm-client:latest docker image, you MUST change another tag.
```
Comment (mrtizmoatwork) [inline src/polyswarmclient/utils.py]: --> "change to another tag."

**Review by mrtizmoatwork — CHANGES_REQUESTED** [review]
windows images are missing `-windows` part of image name.

**Review by alanjds — APPROVED** [review]

**Review by mrtizmoatwork — APPROVED** [review]
ok

**Pull request merged**